app/templates/document_download_template.html

```diff
@@ -39,6 +43,7 @@
 {% block content %}
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">{{ _per_page_title }}</h1>
       {% block main_content %}{% endblock %}
```

Comment:
This block needs to be wrapped in `<div class="govuk-body"></div>`.
```suggestion
      <div class="govuk-body">
        {% block main_content %}{% endblock %}
      </div>
```

Reply:
@nickcolley was saying here (https://github.com/alphagov/document-download-frontend/pull/53#discussion_r374058495) that the `govuk-body` class should be on every `<p>`, not on a wrapping `<div>`. Can we fix as a separate PR?

Reply:
I didn't know that was a rule. I'd be happy to do it later if it is.

Reply:
Meant to say, those paragraphs need it on either them or their parent in order to get the transport font.

Reply:
OK, let’s do it the right way as part of this PR then. Added in 
d790b7d